README.md

```diff
@@ -1,15 +1,15 @@
 # AWS PubSub for Laravel
 
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/pod-point/laravel-aws-pubsub.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-aws-pubsub)
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/pod-point/laravel-aws-pubsub/run-tests?label=tests)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/pod-point/laravel-aws-pubsub/run-tests.yml?branch=main&label=0.0.x)
 [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.md)
 [![Total Downloads](https://img.shields.io/packagist/dt/pod-point/laravel-aws-pubsub.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-aws-pubsub)
 
 **The Pub**
 
-Similar to [Pusher](https://laravel.com/docs/8.x/broadcasting#pusher-channels), this package provides [Laravel Broadcasting](https://laravel.com/docs/8.x/broadcasting) drivers for [AWS SNS](https://aws.amazon.com/sns/) (Simple Notification Service) and [AWS EventBridge](https://aws.amazon.com/eventbridge/) in order to publish server-side events.
+Similar to [Pusher](https://laravel.com/docs/master/broadcasting#pusher-channels), this package provides [Laravel Broadcasting](https://laravel.com/docs/master/broadcasting) drivers for [AWS SNS](https://aws.amazon.com/sns/) (Simple Notification Service) and [AWS EventBridge](https://aws.amazon.com/eventbridge/) in order to publish server-side events.
```

Comment:
Appears that following links to `master` displays versions not yet released - suggest we remove the version entirely as it'll redirect the current stable release automatically, i.e. `https://laravel.com/docs/broadcasting`

<img width="1254" alt="Screenshot 2023-02-07 at 12 23 14" src="https://user-images.githubusercontent.com/17082749/217243981-e1e9be8c-215a-4975-8001-df951e77e275.png">